ginclude/stddef.h

```diff
@@ -18,6 +18,8 @@ typedef int wchar_t;
 
 /* A null pointer constant.  */
 
+/* in case <stdio.h> has defined it. */
```

Comment:
maybe put this comment after `#undef NULL`, on the the same line

Reply:
my text editor didn't like it when I did that.

Reply:
ok. what if you use // style comments?

Reply:
or just remove the comment. putting it above both lines just makes it confusing IMO